tensorrt_llm/_torch/auto_deploy/transform/library/fuse_rope_mla.py

```
@@ -0,0 +1,499 @@
+    permutation so projected data arrives in GPTJ layout at runtime,
+    eliminating the need for a per-step runtime permutation.
+
+    Handles tensor-parallel (TP) column sharding on dim-0.  With TP,
+    ``q_b_proj`` is split by heads and ``kv_a_proj_with_mqa`` is evenly
+    split so its PE rows may only reside on a subset of ranks.
+
+    Returns the number of weight tensors modified.
+    """
+    try:
+        model_config, _ = factory._get_model_config()
+    except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)